roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/svc-ingress-nginx.yml.j2

```diff
@@ -7,6 +7,10 @@ metadata:
   labels:
     app.kubernetes.io/name: ingress-nginx
     app.kubernetes.io/part-of: ingress-nginx
+{% if ingress_nginx_service_annotations %}
+  annotations:
+    {{ ingress_nginx_service_annotations | to_nice_yaml(indent=2, width=1337) | indent(width=4) }}
```

Comment:
Can't the indent be specified directly in `to_nice_yaml` ?

Reply:
Honestly, I copied it from [here](https://github.com/kubernetes-sigs/kubespray/blob/6b0f63a34804c74351f1d73ffd792bb7ecdd62b7/roles/kubernetes-apps/registry/templates/registry-svc.yml.j2#L13). But apparently if I exclude the last indent processor it only indents the first key

Reply:
Ok, so there is a reason for that stuff all around in the codebase ^.  Works for me